src/index.html

```diff
@@ -1,14 +1,82 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis
+    .com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="images/logo.png">
+        <img
+          src="images/logo.png"
+          alt="moyo-logo"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav__item">
+            <a href="#1"class="nav-link is-active">
```

Comment:
There should be spaces between attributes, check this in all code
```suggestion
            <a href="#1" class="nav-link is-active">
```